src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -673,4 +593,16 @@ public static String boldHTML(String input, String alternative) {
         return "<b>" + input + "</b>";
     }
 
+    /**
+     * Quotes each and every character, e.g. '!' as &#33;. Used for verbatim
+     * display of arbitrary strings that may contain HTML entities.
+     */
+    public static String quoteForHTML(String toQuote) {
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < toQuote.length(); ++i) {
+            result.append("&#").append((int) toQuote.charAt(i)).append(';');
```

Comment:
I just looked a bit more and saw that the methods purpose is to encode chars into html and fount out that there is a simpler method from apache commons lang (StingEscapeUtils):
http://stackoverflow.com/questions/1265282/recommended-method-for-escaping-html-in-java